src/services/homework/hooks/submissions.js

```diff
@@ -231,23 +231,20 @@ const noDuplicateSubmissionForTeamMembers = (context) => {
 			newTeamMembers = newTeamMembers.filter((teamMember) => !context.data.submission.teamMembers.includes(teamMember.toString()));
 		}
 
-		let toRemove = '';
 		const submissionsForTeamMembers = context.data.submissions.filter((submission) => {
 			for (let i = 0; i < newTeamMembers.length; i += 1) {
 				const teamMember = newTeamMembers[i].toString();
 				if (submission.teamMembers.includes(teamMember)
 						|| (((submission.studentId || {})._id || {}).toString() == teamMember)
 				) {
-					toRemove += (toRemove == '') ? '' : ', ';
-					toRemove += `${submission.studentId.firstName} ${submission.studentId.lastName}`;
 					return true;
 				}
 			}
 			return false;
 		});
 		if (submissionsForTeamMembers.length > 0) {
 			return Promise.reject(new Conflict({
-				message: `${toRemove + ((submissionsForTeamMembers.length == 1) ? ' hat' : ' haben')} bereits eine Lösung abgegeben! Entferne diese${(submissionsForTeamMembers.length == 1) ? 's Mitglied!' : ' Mitglieder!'}`,
+				message: `${submissionsForTeamMembers.length + ((submissionsForTeamMembers.length === 1) ? ' Mitglied hat' : ' Mitglieder haben')} bereits eine Lösung abgegeben!`,
```

Comment:
Finally there is a solution missing that pupils see that a task is already handed in and from who, while removing this from log is good :)

Reply:
The solution was never there, because the server message is ignored on the client side and there is always a generic message shown - "Submission could not be saved". Could you please be more specific on what the expectation is? Shall I revert the message?

Reply:
Is there any i18 support on the server?

Reply:
as this is not shown in the client, if possible make this visible in the client too and check how students can see if a task is submitted, and if so from who. but this is a follow up task.

Reply:
It is hard for me to judge - is there any UX/UI concept? Do I need to ask Konstantin, how it should work?

Reply:
As discussed with Konstantin - we should keep the generic error message for the moment - no ticket necessary. The ticket will be created if the are any further requirements to this feature.